codegen/header.go

```diff
@@ -1,19 +1,23 @@
 package codegen
 
 import (
+	"goa.design/goa/v3/expr"
 	goa "goa.design/goa/v3/pkg"
 )
 
 // Header returns a Go source file header section template.
-func Header(title, pack string, imports []*ImportSpec) *SectionTemplate {
+func Header(title, pack string, imports []*ImportSpec, meta expr.MetaExpr) *SectionTemplate {
+	value, ok := meta.Last("goa:version:disable")
```

Comment:
`Meta("goa:version:generate", "false")` might be better.
It's a similar syntax to `Meta("openapi:generate", "false").`

Reply:
Done